ffi/capi/examples/wasm/main.mjs

```diff
@@ -0,0 +1,9 @@
+import { FixedDecimal, BufferWritable } from "../../wasm-lib/high-level.mjs"
```

Comment:
Suggestion (non-blocking, can do later): Set up ESLint for this project so that we can enforce code style the same way we do with rustfmt.